tracer/test/Datadog.Trace.TestHelpers/CustomTestFramework.cs

```diff
@@ -117,6 +117,19 @@ protected override async Task<RunSummary> RunTestCollectionsAsync(IMessageBus me
                     })
                     .ToList();
 
+                var seed = new Random().Next();
+
+                DiagnosticMessageSink.OnMessage(new DiagnosticMessage($"Using seed {seed} to randomize tests order"));
+
+                var random = new Random(seed);
+
+                Shuffle(collections, random);
```

Comment:
> Note that the order of the collections themselves isn't changed.

Really? 🤔

Reply:
Oh wow I thought I didn't 😅  It's been too long